studies/templates/studies/study_edit.html

```diff
@@ -118,7 +150,7 @@ <h1 id="manage-researchers" class="panel-title">Manage Researchers</h1>
                 {% if messages %}
                     {% for message in messages %}
                         {% if message.extra_tags == 'user_added' or message.extra_tags == 'user_removed' %}
-                            <div id="add-researcher-messages" class="alert alert-success alert-dismissable">
+                            <div id="add-researcher-messages" class="alert {{ message.tags }} alert-dismissable">
```

Comment:
That's what the message.tags was for.  Before it could only show success messages, now it can show any type.